python/pyspark/sql/tests/test_types.py

```diff
@@ -968,6 +968,19 @@ def __init__(self, **kwargs):
             with self.assertRaises(exp, msg=msg):
                 _make_type_verifier(data_type, nullable=False)(obj)
 
+    @unittest.skipIf(sys.version_info[:2] < (3, 6), "Create Row without sorting fields")
+    def test_Row_without_field_sorting(self):
```

Comment:
no big deal but can we `test_Row_without_field_sorting` -> `test_row_without_field_sorting`? Strictly it follows pep8 I guess (and I personally don't like such names in the current codebase ... )